sentry-kotlin-multiplatform/build.gradle.kts

```diff
@@ -177,14 +177,8 @@ kotlin {
             macosArm64()
         ).forEach {
             it.compilations.getByName("main") {
-                cinterops.create("Sentry.NSException") {
-                    includeDirs("$projectDir/src/nativeInterop/cinterop/SentryNSException")
-                }
-                cinterops.create("Sentry.Scope") {
-                    includeDirs("$projectDir/src/nativeInterop/cinterop/SentryScope")
-                }
-                cinterops.create("Sentry.PrivateSentrySDKOnly") {
-                    includeDirs("$projectDir/src/nativeInterop/cinterop/SentryPrivateSentrySDKOnly")
+                cinterops.create("Sentry.Internal") {
```

Comment:
nit: I think we can remove the workaround below (line 188) now, because we don't include those headers anymore, wdyt?

Reply:
we still do, the cocoapods plugin is still in use so it also pulls in `SentryMechanismMeta`

I decided to keep using cocoapods because of several reasons:
- When I started 'rebuilding' a couple classes in custom objc headers such as SentryEvent I noticed how many classes we actually need and at some point we would maybe shave off 4-5 files if we did it all by hand, which is not a lot.
- The total size of the klib file which contains the bindings that were created by the cocoapods plugin is around 70kb 
- Future maintenance just sounds awful when we have to manually add/remove stuff when things are deprecated or added and it's easy to make a mistake thus crashing the users app

What I did instead: combine all the cinterop libraries that we had into one `SentryInternal` and I shaved off a lot of objc headers that wasn't needed

Reply:
ah ok, gotchu 👍